assets/custom.css

```
@@ -1,8 +1,367 @@
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+Modifications are licensed under
+
+Apache License, Version 2.0
+(see ./LICENSE for details)
+
+*/
+
+:root {
+  --blue-bright: #DDDDDD;
+  --blue-bright: #03B8FF;
+  --blue-light: #2A7DE1;
+  --blue-dark: #074C91;
+  --red-light: #C70039;
+  --red-dark: #900C3F;
+  --orange: #FF7006;
+  --teal-light: #06ECDC;
+  --teal-dark: #17BEBB;
+  --teal-darker: #008C82;
+  --grey-lighter: #EEEEEE;
+  --grey-light: #DDDDDD;
+  --grey: #AAAAAA;
+  --grey-medium: #777777;
+  --grey-dark: #222222;
+}
+
+body {
+  background-color: #f9f9f9;
+  color: var(--grey-dark);
+  font-size: 1.4rem;
+  margin: 0rem;
+  overflow: hidden;
+}
+
+hr {
+  margin: 0.3rem 0rem;
+}
+
+#description-card {
+  width: 100%;
+  margin: 2rem 0rem;
+}
+
+#banner {
+  height: 5.5rem;
+  background-color: var(--blue-dark);
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+#banner img {
+  position: relative;
+  height: 3rem;
+  width: auto;
+  margin-left: 2rem;
+}
+
+h1, h2, h3, h4, h5, h6, td, th, span, a, p, label {
+  font-family: "Helvetica Neue", sans-serif;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  color: var(--blue-dark);
+  font-weight: 400;
+}
+
+h1 {
+  font-size: 3rem;
+  margin-top: 3rem;
+}
+
+h3 {
+  font-size: 2.5rem;
+  margin: 2.5rem 0 1rem;
+}
+
+h4 {
+  font-size: 2rem;
+  margin-top: 2rem;
+  font-weight: 500;
+}
+
+h4.table-label {
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: self-end;
+}
+
+h4.table-label span {
+  font-size: 1.8rem;
+}
+
+label {
+  margin: 2rem 0 0.2rem;
+  font-weight: 500;
+}
+
+input[type="checkbox"] {
+  margin: 0 1rem 0 0;
+}
+
+#columns {
+  display: flex;
+  height: 100%;
+}
+
+#left-column {
+  display: flex;
+}
+
+#left-column > div > div {
+  padding: 0 2rem;
+  width: 38rem;
+}
+
+#left-column > div:first-child {
+  width: 42rem;
+  height: calc(100vh - 5.5rem);
+  transition: width 0.6s ease-in-out;
+  overflow-y:auto;
+  overflow-x:hidden;
+}
+
+#left-column.collapsed > div:first-child {
+  width: 0;
+}
+
+#right-column {
+  background-color: var(--grey-light);
+  padding: 2rem 0 0;
+  width: 100%;
+  overflow-x: auto;
+}
+
+.dash-table-container {
+  /* viewport height - (tab + banner height) */
+  height: calc(100vh - 14.3rem);
+  overflow-y: auto;
+}
+
+.rc-slider-handle,
+.rc-slider-dot-active {
+    border-color: var(--blue-light);
+}
+
+.rc-slider-handle:active,
+.rc-slider-handle:hover,
+.rc-slider-handle-click-focused,
+.rc-slider-handle-click-focused:focus {
```

This is just removing the blue hover effect when it's disabled 

```suggestion
.rc-slider:not(.rc-slider-disabled) .rc-slider-handle:active,
.rc-slider:not(.rc-slider-disabled) .rc-slider-handle:hover,
.rc-slider:not(.rc-slider-disabled) .rc-slider-handle-click-focused,
.rc-slider:not(.rc-slider-disabled) .rc-slider-handle-click-focused:focus {
```